10/umbraco-deploy/release-notes.md

```diff
@@ -186,6 +194,14 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
 <details>
 <summary>Version 4</summary>
 
+#### [4.10.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.10.0) (March 19th 2024)
+
+* All items from 4.10.0-rc1
+* Fixed issue with transfer of date values within Nested Content, Block List or Block Grid properties [#209](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/209)
+* Fixed issue where templates could incorrectly cause schema mismatch errors when running in production mode [#187](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/187)
```

Comment:
A quick note on this description: CMS v8 doesn't have a production runtime mode, but we've backported this fix to Deploy 4 to ensure the template UDA files are correctly processed (by always setting the template path).

Reply:
Have updated the description here.